apps/vs-code-designer/src/app/commands/workflows/switchToDotnetProject.ts

```diff
@@ -254,6 +255,10 @@ async function getArtifactNamesFromProject(target: vscode.Uri): Promise<Record<s
       continue;
     }
 
+    if (file === parametersFileName) {
+      artifactDict['connections'].push(parametersFileName);
```

Comment:
Is using 'connections' not going to cause a key collision in the dictionary overriding where 'connections' is already used?

Reply:
Yeah thats right, @mireedmsft  You might need to change to 'parameters'.

Reply:
So in this case artifactDict is a dictionary of string to array, so adding the parameters file here just adds another element to the array at that index (and updateBuildFile already assumes that each element is an array and iterates through)